cli/cli_util.c

```diff
@@ -301,16 +245,21 @@ kvdb_compact_request(const char *kvdb_home, const char *request_type, u32 timeou
             goto err_out;
         }
 
-        rest_status_yaml(&status, stat_buf, sizeof(stat_buf));
-
-        printf("%s", stat_buf);
+        printf(
+            "samp_lwm: %d\n"
+            "samp_hwm: %d\n"
+            "samp_curr: %d\n"
+            "active: %s\n"
+            "canceled: %s\n",
+            status.kvcs_samp_lwm, status.kvcs_samp_hwm, status.kvcs_samp_curr,
+            status.kvcs_active ? "true" : "false", status.kvcs_canceled ? "true" : "false");
```

Comment:
didn't this used to print in floating point?

Reply:
Yes, would you like me to revert back?